sycl/doc/extensions/GroupMask/GroupMask.asciidoc

```diff
@@ -286,25 +285,24 @@ struct group_mask {
   void flip();
   void flip(id<1> id);
 
-  bool operator==(group_mask rhs) const;
-  bool operator!=(group_mask rhs) const;
+  bool operator==(const group_mask& rhs) const;
+  bool operator!=(const group_mask& rhs) const;
 
-  group_mask operator &=(group_mask rhs);
-  group_mask operator |=(group_mask rhs);
-  group_mask operator ^=(group_mask rhs);
-  group_mask operator <<=(size_t);
-  group_mask operator >>=(size_t rhs);
+  group_mask &operator &=(const group_mask& rhs);
+  group_mask &operator |=(const group_mask& rhs);
+  group_mask &operator ^=(const group_mask& rhs);
+  group_mask &operator <<=(size_t n);
+  group_mask &operator >>=(size_t n);
```

Comment:
None of these changes are reflected in the table that describes the member functions.  The sample header and the table should be in sync.

Reply:
fixed